components/protocomm/include/transports/protocomm_ble.h

```diff
@@ -28,7 +28,11 @@ extern "C" {
 #define BLE_UUID128_VAL_LENGTH  16
 
 typedef enum {
+    // Peer is connected without any security (i.e., no PoP has been entered)
     PROTOCOMM_BLE_PEER_CONNECTED,
+    // Peer is connected securely with a valid PoP
+    PROTOCOMM_BLE_PEER_CONNECTED_SECURE,
```

Comment:
it's generally a good idea to put new enumerators at the end, so that the values of the existing enumerators stay the same